libraries/stdlib/samples/test/samples/ranges/ranges.kt

```diff
@@ -52,4 +52,34 @@ class Ranges {
         assertTrue(3.14f in range)
         assertFalse(100.1f in range)
     }
+
+    @Sample
+    fun intStep() {
+        val range = 1..6 step 2
```

Comment:
Makes sense also to show how to create a downwards-iterating progression by applying `step` to the result of `downTo`.

Reply:
> Makes sense also to show how to create a downwards-iterating progression by applying `step` to the result of `downTo`.

added